src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -49,8 +51,21 @@ export const DatasetSearchTable = (props) => {
   const [selectedTable, setSelectedTable] = useState(datasetSearchTableTabs.study);
   const [searchTerm, setSearchTerm] = useState('');
 
-  const isFiltered = (filter, category) => (filters[category]).indexOf(filter) > -1;
-  const numSelectedFilters = (filters) => Object.values(filters).reduce((sum, array) => sum + array.length, 0);
+  const isFilteredArray = (filter, category) => (filters[category]).indexOf(filter) > -1;
+
+  const numSelectedFilters = (filters) => {
+    var sum = 0;
+    for (const category in filters) {
+      if (isArray(filters[category])) {
+        sum += filters[category].length;
+      } else {
+        if (filters[category]) {
```

Comment:
I would combine an `if` after an `else` here
```suggestion
      } else if (filters[category]) {
```